doc/modules/covariance.rst

```diff
@@ -333,10 +336,12 @@ attributes of a :class:`MinCovDet` robust covariance estimator object.
 .. |robust_vs_emp| image:: ../auto_examples/covariance/images/sphx_glr_plot_robust_vs_empirical_covariance_001.png
    :target: ../auto_examples/covariance/plot_robust_vs_empirical_covariance.html
    :scale: 49%
+   :alt:
```

Comment:
:alt: Two line charts comparing the estimation errors that are made when using various types of location and covariance estimates on contaminated Gaussian distributed data sets.